src/main/resources/static/js/gw.history.js

```diff
@@ -205,7 +205,39 @@ GW.history = {
      */
     getProcessHistoryTable: function(msg){
 
-        var content = `<table class=\"table table-color\" id=\"process_history_table\"> 
+        let content = `
+        <div class="history-filters" style="display: flex; margin: 10px; justify-content: space-between;">
+            <div id="recentDaysFilterContainer">
+                <label for="recentDaysFilter">Recent Days:</label>
```

Comment:
The other two filters make sense. But this one is a little confusing. The current records are aligned based on the time. People can see the recent records directly. No need to filter using last n days here. Can you remove?

Reply:
ok sure